app/models/concerns/nc_residence_county_concern.rb

```diff
@@ -0,0 +1,53 @@
+# Handles the bits of the North Carolina state file intake that deals with the
+# county.
+module NcResidenceCountyConcern
```

Comment:
A little pre-emptive but I expect that our NCIntake is going to become huge over time so I went ahead and broke this bit out. I can move it into the intake instead if it is felt strongly.